stacks/dotnet50/devfile.yaml

```diff
@@ -25,6 +25,7 @@ components:
 - name: dotnet
   container:
     image: registry.access.redhat.com/ubi8/dotnet-50:5.0
+    command: ["tail", "-f", "/dev/null"]
```

Comment:
This is also widely used in the old Che devfile v1 images, e.g. https://github.com/eclipse-che/che-devfile-registry/blob/main/dockerfiles/entrypoint.sh

Reply:
@amisevsk thanks I will test that.

Reply:
After using `args` instead of `command` these stacks are still failing:

```
nodejs-nextjs nodejs java-quarkus php-laravel go nodejs-nuxtjs nodejs-vue python java-websphereliberty dotnet50 nodejs-react dotnet60 nodejs-svelte java-websphereliberty-gradle java-openliberty-gradle java-openliberty nodejs-angular dotnetcore31
```

Reply:
Here's a summary of the entrypoint for each image:

|STACK NAME                    |IMAGE                                                           |ENTRYPOINT|
|-|-|-|
|nodejs-nextjs                 |node:lts-slim                                                   |["docker-entrypoint.sh"]|
|nodejs                        |registry.access.redhat.com/ubi8/nodejs-14:latest                |["container-entrypoint"]|
|java-quarkus                  |registry.access.redhat.com/ubi8/openjdk-11                      |null|
|php-laravel                   |composer:2.1.11                                                 |["/docker-entrypoint.sh"]|
|go                            |golang:latest                                                   |null|
|nodejs-nuxtjs                 |node:lts                                                        |["docker-entrypoint.sh"]|
|nodejs-vue                    |node:lts-slim                                                   |["docker-entrypoint.sh"]|
|python                        |quay.io/eclipse/che-python-3.7:nightly                          |["/entrypoint.sh"]|
|java-websphereliberty         |icr.io/appcafe/websphere-liberty-devfile-stack:22.0.0.1         |null|
|dotnet50                      |registry.access.redhat.com/ubi8/dotnet-50:5.0                   |["container-entrypoint"]|
|nodejs-react                  |node:lts-slim                                                   |["docker-entrypoint.sh"]|
|dotnet60                      |registry.access.redhat.com/ubi8/dotnet-60:6.0                   |["container-entrypoint"]|
|nodejs-svelte                 |node:lts-slim                                                   |["docker-entrypoint.sh"]|
|java-websphereliberty-gradle  |icr.io/appcafe/websphere-liberty-devfile-stack:22.0.0.1-gradle  |null|
|java-openliberty-gradle       |icr.io/appcafe/open-liberty-devfile-stack:22.0.0.1-gradle       |null|
|java-openliberty              |icr.io/appcafe/open-liberty-devfile-stack:22.0.0.1              |null|
|nodejs-angular                |node:lts-slim                                                   |["docker-entrypoint.sh"]|
|dotnetcore31                  |registry.access.redhat.com/ubi8/dotnet-31:3.1                   |["container-entrypoint"]|

Checking a few of these, it seems they should be fine if `args` is set to `["tail", "-f", "/dev/null"]` and `command` is left unchanged (e.g. the node:lts and node:lts-slim images use an entrypoint that should just `exec` the args) -- running `docker run -it --rm $image tail -f /dev/null` does result in a non-terminating container.

I don't know if `odo` is doing anything fancy for these images, but if it was always just overriding the command/args then it _should_ work as far as I can tell.